### DISCUSSION TITLE
Adjust Stacked Bar Sizing and Y-axis

### DISCUSSION THREAD

**Discussion by shiftyoliver** [EvotecIT/PSWriteHTML#372] [General]
Hi, is there a way to avoid the autosizing width of the stacked bar chart as you add more names and instead maintain one set width? Also can you adjust the y-axis scale and the maximum value for it? I do see a Width parameter for new-HTMLchart but it only accepts type object, what kind of object is it looking for?

**Comment by shiftyoliver** [comment]
I'd like to clarify, meaning the width of the bars themselves. When I add around more then 20 names everything gets scrunched too small and the bars are to thin to make it easy to view.

**Reply by PrzemyslawKlys** [reply]
You gotta help me out and provide some examples, showing a problem.
